lists/views.py

```diff
@@ -33,7 +33,7 @@ def edit_list(request, id):
         list = List.objects.get(id=id)
         #form = EditListForm()
         form = EditListForm(instance=list)
-        return render(request, 'edit_list.html', {'list':list}, {'form':form})
+        return render(request, 'edit_list.html', {'list':list,'form':form})
```

Comment:
If you don't want any of this except for the fix, its this line

Reply:
i spent more hours than im willing to say looking for this mistake

Reply:
rip